packages/docs/docs/integration/health-gorilla.md

```diff
@@ -11,7 +11,7 @@ Health Gorilla is an interoperability services used to order labs and diagnostic
 This is advanced integration. [Email us](mailto:info+healthgorilla@medplum.com?subject=Health%20Gorilla%20Integration%20for%20Medplum) for more information.
 :::
 
-The Medplum toolkit provides tooling to optimize the physician ordering experience (CPOE), to minimize errors and enhance patient experience. A well defined workflow with diagnostic menu will help set you up for success. The workflows shown here are intended to reduce the amount of repeated data entry required by clinicians and staff.
```

Comment:
I noticed it said CPOE here but it was missing the "C". I attempted to keep the same spirit of what the sentence was initially going for. Happy to delete PR if this was not the intent.